app/components/login/Login.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { createClass, Component } from 'react';
+
+const Login = createClass({
+
+  signUserIn(email, password) {
```

Comment:
I don't really know anything about `react-router-dom`. Perhaps you can use this issue I created to help impement `react-router`. https://github.com/teamyouhired/youhired/issues/32